Tests/TuistGeneratorTests/Generator/LinkGeneratorTests.swift

```diff
@@ -88,6 +88,43 @@ final class LinkGeneratorErrorTests: XCTestCase {
             XCTAssertEqual($0 as? LinkGeneratorError, LinkGeneratorError.missingProduct(name: "Test"))
         }
     }
+    
+    func test_generateEmbedPhase_setupEmbedFrameworksBuildPhase_whenXCFrameworkIsPresent() throws {
```

Comment:
@pepibumur @kwridan added tests for `XCFramework's generateEmbedPhase`

Reply:
I have come across a reference to waka file at https://github.com/tuist/tuist/blob/8635579dbdd3bd21d5bcf377bed4ef8b84ae6e50/Tests/TuistGeneratorTests/Generator/LinkGeneratorTests.swift#L45, i have tried using the approach as done in `LinkGeneratorTests` at aforementioned link but it doesn't work for me.

Reply:
I have managed to add test assertion for `PBXFileReference` added in the build phase during  `generateEmbedPhase` and updated the test case, I was incorrectly referencing `product` instead of `file`, got `wakaFile` working now